Plugins/PluginUtilities/PluginUtilities.cpp

```diff
@@ -938,6 +938,100 @@ Quaternion HkMatrixToQuaternion(Matrix m)
 	return quaternion;
 }
 
+Matrix TransposeMatrix(Matrix& m)
+{
+	return { {
+		{m.data[0][0], m.data[1][0], m.data[2][0]},
+		{m.data[0][1], m.data[1][1], m.data[2][1]},
+		{m.data[0][2], m.data[1][2], m.data[2][2]}} };
+}
+
+float MatrixDeterminant(Matrix& m, uint row, uint col)
```

Comment:
This is actually like... MatrixSubDeterminant or something. It works on the 2x2 matrices that form a 3x3 matrix, ignoring the rows and columns identified by `row` and `col` and taking the remaining values as a 2x2 matrix